src/components/Navigator/NavigatorCardItem.vue

```diff
@@ -10,7 +10,6 @@
 
 <template>
   <BaseNavigatorCardItem
-    class="navigator-card-item"
```

Comment:
Why are we removing this?

Reply:
The class is duplicated on BaseNavigatorCardItem that's why I deleted it